remove-boolean.patch

```diff
@@ -0,0 +1,145 @@
+diff --git a/ADQL.tex b/ADQL.tex
```

Comment:
Was this file intended to be committed?

Reply:
Yes, it was. I just want to keep somewhere what was did to support boolean constraint. So, yes, I could use the Git history for that, but I would have to browse/search in the history. With this easy and simple patch I do not have to do so. But if you think it is useless, I keep it only locally on my computer.